glue_jupyter/state_traitlets_helpers.py

```diff
@@ -74,7 +74,7 @@ def update_state_from_dict(state, changes):
                 if changes[name] != MAGIC_IGNORE and getattr(state, name) != changes[name]:
                     if name == 'cmap':
                         if changes[name] != state.cmap.name:
-                            setattr(state, name, get_cmap(changes[name]))
+                            setattr(state, name, matplotlib.colormaps[changes[name]])
```

Comment:
```suggestion
                            setattr(state, name, colormap.get_cmap(changes[name]))
```
Probably better to stick with the function – although the `state.cmap.name` suggests this will always be called in the string form, `get_cmap` supports all three possible input types, although direct lookup might be faster.
For `colormaps[changes[name]]` to work the legacy import above would have to be changed to
```
from matplotlib.cm import cmaps_listed as colormaps
```
(but then would provide no `get_cmap` at all).